src/site/_includes/partials/translation-disclaimer.njk

```diff
@@ -1,12 +1,9 @@
 {% set gitlocalizeUrl =  page.inputPath | gitlocalizeLink(lang) %}
-{% if translation %}
+{% if translated %}
 {% Banner 'neutral' %}
-    {% if translation === 'none' %}
+    {% if translated === 'none' %}
```

Comment:
Since you're removing the third option of `translated === 'machine'` what do you think about making `translated` into a boolean?

Reply:
I'd like to have a date in "translated" field in order to be able to compare to "updated" field in the original post, e.g. to find stale translations.